debug/ExplainAction.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace yii\mongodb\debug;
+
+use yii\base\Action;
+use yii\base\InvalidParamException;
+use yii\debug\panels\DbPanel;
+use yii\helpers\ArrayHelper;
+use yii\helpers\Json;
+use yii\web\HttpException;
+
+/**
+ * ExplainAction provides EXPLAIN information for MongoDB queries
+ */
+class ExplainAction extends Action
+{
+    /**
+     * @var DbPanel
+     */
+    public $panel;
+
+    /**
+     * Runs the action
+     *
+     * @return string result content
+     */
+    public function run($seq, $tag)
+    {
+        $this->controller->loadData($tag);
+
+        $timings = $this->panel->calculateTimings();
+
+        if (!isset($timings[$seq])) {
+            throw new HttpException(404, 'Log message not found.');
+        }
+
+        $query = $timings[$seq]['info'];
+        preg_match('/^.+\((.*)\)$/', $query, $matches);
```

Comment:
@samdark I need to get a substring in parentheses, so your version does not suit you.